tests/ProxyManagerTest/Functional/LazyLoadingGhostFunctionalTest.php

```diff
@@ -829,4 +830,109 @@ public function getPropertyAccessProxies()
             ],
         ];
     }
+
+    /**
+     * @dataProvider skipPropertiesFixture
+     *
+     * @param string   $className
+     * @param string   $propertyName
+     * @param string   $expected
+     * @param string[] $proxyOptions
+     */
+    public function testSkipProperties($className, $propertyName, $expected, $proxyOptions)
+    {
+        $proxy = $this->generateProxy($className, $proxyOptions);
+        $ghostObject = new $proxy(
```

Comment:
You are still using the constructor instead of `$proxy::staticProxyConstructor()` :P